playbooks/update_files.yml

```diff
@@ -146,6 +146,25 @@
             __present_python_templates + __present_shellcheck_templates)) +
             ([__role_template_dir] | product(__role_present_templates)) }}"
 
+        - name: Ensure badges in README.md
+          lineinfile:
+            path: "{{ git_dir }}/README.md"
+            insertafter: '^# \w'
+            firstmatch: true
+            line: "\n \
+              {%- if 'python_roles' in group_names -%}\
+              ![CodeQL](https://github.com/linux-system-roles/{{ inventory_hostname }}/workflows/CodeQL/badge.svg) \
```

Comment:
It would be cool if we could somehow associate each badge with an action so that we didn't have this hardcoded list here.
The journald role has all of the badges in the title line - see https://github.com/linux-system-roles/journald/blob/main/README.md

Reply:
Not sure what you meant under "associate each badge with an action", but I found a way to automatically generate these badges from existing workflow, excluding some of them that do not require a badge. And also added `present_badges` in roles' host_vars to define role-specific badges. Tested it with network so far. Please take a look.